.changeset/funny-rivers-divide.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react-brand': patch
+---
+
+Updating the colors documentation to remove rgba and hex values
```

Comment:
We won't need a changeset for this update, as there's no change in the published package. Sorry, should have mentioned that earlier 🙈

Reply:
I removed it